godel/config/license-plugin.yml

```diff
@@ -12,3 +12,33 @@ header: |
   // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   // See the License for the specific language governing permissions and
   // limitations under the License.
+custom-headers:
+  - name: go-rsa-2010
+    header: |
+      // Copyright 2010 The Go Authors. All rights reserved.
+      // Use of this source code is governed by a BSD-style
+      // license that can be found in the LICENSE file.
+      //
+      // Copyright (c) {{YEAR}} Palantir Technologies. All rights reserved.
+      //
```

Comment:
Also a nit: we can drop this newline I think and just have all 3 lines together like the Google one.